numpy/lib/npyio.py

```diff
@@ -475,7 +475,7 @@ def save(file, arr, allow_pickle=True, fix_imports=True):
 
     Notes
     -----
-    For a description of the ``.npy`` format, see :py:mod:`numpy.lib.format`
+    For a description of the ``.npy`` format, see :ref:`binary-serialization`
```

Comment:
What about the references in `savez` and `savez_compressed`?

Reply:
Hmm, also need periods at the ends of the sentences in all the places.

Reply:
fixed, thanks